www/content/docs.md

```diff
@@ -913,25 +913,32 @@ on the client side.
 
 htmx includes a number of useful headers in requests:
 
-| Header | Description
-|--------|--------------
-| `HX-Request` | will be set to "true"
-| `HX-Trigger` | will be set to the id of the element that triggered the request
-| `HX-Trigger-Name` | will be set to the name of the element that triggered the request
-| `HX-Target` | will be set to the id of the target element
-| `HX-Prompt` | will be set to the value entered by the user when prompted via [hx-prompt](@/attributes/hx-prompt.md)
+| Header | Description |
+|--------|-------------|
+| `HX-Boosted` | indicates that the request is via an element using [hx-boost](@/attributes/hx-boost.md)
+| `HX-Current-URL` | the current URL of the browser
+| `HX-History-Restore-Request` | `true` if the request is for history restoration after a miss in the local history cache
```

Comment:
This and the other "true" should be in quotation marks since they're strings, not variables. Everything else seems good.

Reply:
just copied what was in reference.md so I updated both files